Tools/CheckCmdletXmlDocs.psm1

```diff
@@ -125,31 +126,33 @@ function Check-CmdletDoc() {
             (-not (InSpecifiedCloudProducts $CloudProducts $productMapping $apiMappings))) {
                 continue
         }
-
-        Write-Host "`nChecking $($cmdlet.Name)..."
         
         # Get the cmdlet's documentation.
         $docObj = Get-Help -Full $cmdlet.Name
 
         # Check the documentation for important information/categories and write relevant warnings.
-        $wroteWarnings = WriteAllFieldWarnings $docObj $productMapping.Value $outputWhitelist
+        $warnings = @()
+        $warnings += (GetAllFieldWarnings $docObj $productMapping.Value $outputWhitelist)
 
         # If there are examples, and the user has chosen a DeepExampleCheck, check if the examples include a command
         # starting with the usual PS C:\>. 
         # If they do, check that they also have an intro and sample output for the command.
         if ($DeepExampleCheck) {
-            $wroteWarnings = ((DoDeepExampleCheck $docObj) -or $wroteWarnings)
+            $warnings += (DoDeepExampleCheck $docObj)
         }
 
         # Check that all parameters have a valid (non-null, non-whitespace) description. 
-        $wroteWarnings = ((WriteParameterDescriptionWarnings $docObj) -or $wroteWarnings)
+        $warnings += (GetParameterDescriptionWarnings $docObj)
 
-        if (-not ($wroteWarnings)) {
-            Write-Host "PASSED" -ForegroundColor "Green" -BackgroundColor "Black"
+        if ($warnings.Count -gt 0) {
+            Write-Host "`n$($cmdlet.Name)" -ForegroundColor Red -BackgroundColor Black
+            $warnings | Write-Warning
+        } else {
+            $numPassedCmdlets++
         }
     }
 
-    Write-Host
+    Write-Host "`nNumber of Passing Cmdlets: $numPassedCmdlets`n" -ForegroundColor Green -BackgroundColor Black
```

Comment:
Perhaps I wasn't clear. Could we have the "number of passing cmdlets" broken down per product? e.g.:
```
Checking "Google Cloud Storage cmdlets"
Warning XXX
Warning XXX
Number of passing cmdlets: 34

Checking "Google Compute Engine"
Warning XXX
...
```

Is that a minor change, or would that be a big pain to add?